Tests/ClientTests/TabManagement/TabManagerTests.swift

```diff
@@ -10,7 +10,6 @@ import Shared
 @testable import Client
 
 class TabManagerTests: XCTestCase {
```

Comment:
I created https://mozilla-hub.atlassian.net/browse/FXIOS-7261 for tests in this class (so we fix them + add others)